### ISSUE TITLE
Accessibility Feedback, Design Intent Collaboration for Debt Letters MVP

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#9685]
# Accessibility Feedback

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

1. For each of the pages, **consider** adding an intro paragraph of 1-3 sentences, between the h1 and the rest of the page, explaining what the content on the page is, and that there is guidance for how to pay. Why: For mobile and screen reader users, the "How to pay" and "Need help?" information will be very far down in the reading order. The stress of reading about the debts may overwhelm, and they may not get to it.

1. **Consider** asking if analytics exist for how users currently access this data. I'm curious how much of their traffic is currently on mobile versus desktop. My hunch is this is content people will access on mobile.

1. The Need help? phone number **should** be a link. This will be a nice way to jump to action immediately using the OS phone support or dialing directly on a mobile device.

1. Any downloadable and printable letters (PDFs?) need to be accessibility audited, **must** be 508 compliant. Do we know where these are going to be coming from? Are they generated from HTML, another application, or linked from another group at VA? If this is generated programmatically with the API, that will have a bearing on the conversation.

1. Really like how the pages are divided up using headings and type hierarchy. This makes scanning easy and accurate.

1. **Consider** making the h1 more specific to the page content, re the Manage your VA debts.

## Definition of Done
- [ ] Acknowledge feedback, communicate decisions made.
- [x] Need help is coded as a phone number. This doesn't need to be updated in designs, unless it's convenient.
- [ ] Provide a quick update when you know where the PDF will be coming from. 

@1Copenut — feedback is ready for your 👀 . Thank you!

**Comment by leahkeeler** [comment]
Jen, F/U question for you - #4 The letters will be downloadable from eFolders/vbms - that is a requirements, the ability to download the letters veterans receive in the mail. If the information that is presented in the cards is the same as contained in the letters, as well as content around next steps, but better written and presented - is there still a requirement around making the actual pdf's 508 compliant which you said is very difficult, since the same information will be presented in .html which is 508 compliant and the ability to download and see the actual letters is very similar to getting the letters printed on paper in the mail - which obviously is not compliant. Thanks, Leah

**Comment by 1Copenut** [comment]
@jenstrickland Feedback looks good, thank you!

**Comment by jenstrickland** [comment]
Hi Leah, yes, the PDFs must be 508 compliant. Blind and low-vision users would have braille letters mailed to them. Here's the Section 508 requirement:

> This law requires that all website content be accessible to people with disabilities, including government hosted or contractor hosted websites and intranet sites, both public- and government-facing. 
> This applies to Web applications, Web pages, and all attached files on the Intranet and the Internet,  including website content, emails, computer software, gaming systems, apps, and PDF versions of traditionally printed assets. PDFs require 508 remediation as they are not accessible by screen readers.

**Comment by rebeccawalsh** [comment]
@jenstrickland 

1. Would something like, "View the details of your VA debt and find out the next steps to resolving them." be enough, or should it have more specificity? I'm sure Peggy will have a rewrite for me, but as far as direction goes?

2. There is no way for users to currently access this data online. We have asked for analytics regarding the debtman site, which is just static content pages.

3. This was a mistake with unformatted copy pasting. All phone numbers and emails will have their respective tel's and mailto's.

4. @mattpointzxer0 @leahkeeler 🔔 SCOPE DISCUSSION! 🔔 

5. Thanks!

6. Yeah, we didn't put too much thought here since we didn't know where it would live. We're considering a couple of options but will likely tag Mikki's expertise and ask what she thinks will be most consistent and clear. They will be distinct.

**Comment by jenstrickland** [comment]
1. Peggy will certainly provide the best guidance. Something along the lines of "A sentence about VA debts overall. Review your VA debts, submit payments, and get help."

1. Thanks! VA.gov analytics indicate about 40% of the traffic is mobile, and I guess that's fine to use since Debtman doesn't have analytics available. 

1. ✔️ 

1. Is it possible to get one of the PDFs soon, to evaluate if it is a lot of work? Depending on how it is generated, we may be able to adjust the process to output better quality. 


Thanks!
